package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "keyboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "script.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix"
+  },
+  "repository": {
+    "type": "git",
```

Comment:
Try to use webpack next time

Reply:
Yes, sure. I've already used webpack in task "English for Kids"